popsicles.py

```diff
@@ -0,0 +1,55 @@
+from datetime import date, timedelta
+
+
+class Popsicles:
+    def __init__(self, price, flavor, color, expiration_date, popularity):
+        self._price = price
+        self._flavor = flavor
+        self._color = color
+        self._expiration_date = expiration_date
+        self._popularity = popularity
+
```

Comment:
Great/ maybe we should have both 'price' which is the price for our customers, and 'cost' which what wer pay to get the popsicle